pr2eus_tutorials/launch/gazebo/pr2_tabletop_sim.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+killall gzserver 2&>1 1>/dev/null || true
+
+set -e
+
+export ROBOT=sim
```

Comment:
@knorth55 do we really need this? 

`roslaunch hoge.launch machine:=c2` for real robot and `roslaunch hoge.launch machine:=loclahost` for simulation is not enough?

Reply:
the simulation demo doesnt work in kinetic, so i need to update and find out what is wrong.
the launch file uses ENV ROBOT inside, so probably it is necessary